src/App.tsx

```diff
@@ -16,48 +19,64 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isSortedBy, setIsSortedBy] = useState('');
```

Comment:
You need 3 types of sort ( A - ALPHABETICAL, L - LENGTH, D - DEFAULT). 
Add it and provide it in this state.